File: fundamentals-of-programming/list-3/exercise-15.py
cliente = int(input("Código do cliente: "))
while(cliente != 0):
  total = 0

  produto = int(input("Código do produto: "))
  while (produto != -1):
    preco = float(input("Preço do produto: "))
    quantidade = int(input("Quantidade do produto: "))
    total += preco * quantidade

    produto = int(input("Código do produto: "))

  print(f"Total da compra: R${total:.2f}")

  cliente = int(input("Código do cliente: "))

File: fundamentals-of-programming/bimester-2/list-1/exercise-15.c
#include <stdio.h>
#include <stdlib.h>

main() {
  int cliente, quantidade, produto;
  float preco, soma;

  do {
    printf("Digite o código do cliente (0 para parar): ");
    scanf("%d", &cliente);

    if (cliente != 0) {
      do {
        printf("Digite o código do produto: ");
        scanf("%d", &produto);

        if(produto != -1) {
          printf("Digite o preço unitário: R$");
          scanf("%f", &preco);

          printf("Digite a quantidade do produto %d: ", produto);
          scanf("%d", &quantidade);

          soma += preco * quantidade;
        }
      } while (produto != -1);

      printf("Total da compra: R$%.2f\n", soma);

      soma = 0;
    }
  } while(cliente != 0);
}